### ISSUE TITLE
noita-proxy-win console is blank and GUI broken

### ISSUE THREAD

**Issue by kitspin** [IntQuant/noita_entangled_worlds#3]
hello! earlier today i downloaded the proxy and installed mod and was able to get it to run fine. now a few hours later, when trying to run the proxy again, the console is blank the GUI doesn't load (it appears in the taskbar but clicking it does nothing). I tried deleting it and re-downloading it, but the same thing happens.
![image](https://github.com/IntQuant/noita_entangled_worlds/assets/159738142/381dc84a-049d-4fc9-997a-daf3970a115c)

**Comment by IntQuant** [comment]
Hi! Try to check if it stays open somewhere in background, and maybe try to
reboot? Sorry, won't be able to respond until morning.

пт, 17 мая 2024 г., 00:49 kitspin ***@***.***>:

**Comment by kitspin** [comment]
Issue persists after reboot, and i didn't see anything in the task manager.

and no worries! just wanted to make you aware of the issue.

**Comment by IntQuant** [comment]
Not sure what to do about this then, egui apps *generally* don't do that, and I'm not sure if I can reproduce it locally.

**Comment by kitspin** [comment]
figured it out! i'm using a 4k tv as a monitor, so my scaling is set to 300%. sometimes that breaks certain programs bc the UI doesn't scale properly. 

for anyone else who has this problem: I was able to fix it by going to the exe's properties > Compatibility > Change high DPI settings > High DPI scaling override > "System (Enhanced)". this works with a number of apps that have this problem.

I have no clue why this only happened after the second time I opened the program.
